addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -601,6 +601,16 @@ def _import_fill_invoice_form(self, journal, tree, invoice_form, qty_factor):
                 invl_logs = self._import_fill_invoice_line_form(journal, invl_el, invoice_form, invoice_line_form, qty_factor)
                 logs += invl_logs
 
+        # ==== total tax (to avoid rounding mismatch) ====
+
+        xml_tax_amount_per_rate = {}
+        for tax_node in tree.findall('.//{*}TaxSubtotal'):
+            rate = tax_node.find('.//{*}Percent').text
+            amount = tax_node.find('.//{*}TaxAmount').text
```

Comment:
it's always a bit dangerous to access `text` like this since a broken xml may not contain the node, hence raising an AttributeError (you could also use `findtext`)